CommandLine.c

```diff
@@ -338,6 +351,8 @@ int CommandLine_run(const char* name, int argc, char** argv) {
 
    if (flags.delay != -1)
       settings->delay = flags.delay;
+   if (flags.maxIterations != -1)
+      settings->maxIterations = flags.maxIterations;
```

Comment:
Why not put this as part of the `ScreenManager` state directly?

Reply:
That can be done too. I never thought of that.